Source/santad/DataLayer/SNTRuleTable.m

```diff
@@ -87,6 +91,31 @@ - (uint32_t)initializeDatabase:(FMDatabase *)db fromVersion:(uint32_t)version {
         self.launchdCertSHA, @(SNTRuleStateWhitelist), @(SNTRuleTypeCertificate)];
   }
 
+  // Setup critical system binaries
+  // TODO(tburgin): Add the Santa components to this feature and remove the santadCertSHA rule.
+  NSMutableDictionary *bins = [NSMutableDictionary dictionary];
+  for (NSString *path in @[ @"/usr/libexec/trustd" ]) {
+    SNTFileInfo *binInfo = [[SNTFileInfo alloc] initWithPath:path];
```

Comment:
Does this fail the right way if there's no file at path?  I'm guessing csInfo will end up nil and then the call to signingInformationMatches: will return NO?